contrib/epee/include/epee/net/abstract_tcp_server2.inl

```diff
@@ -103,7 +103,7 @@ PRAGMA_WARNING_DISABLE_VS(4355)
 		m_local(false),
 		m_ready_to_close(false)
   {
-    MDEBUG("test, connection constructor set m_connection_type="<<m_connection_type);
+    //MDEBUG("test, connection constructor set m_connection_type="<<m_connection_type);
```

Comment:
lots of commented-out (rather than removed) log statements throughout the commit; should probably be removed if no longer relevant?